src/mixins/amalgamation-mixin.ts

```diff
@@ -205,9 +204,8 @@ export default class AmalgamationMixin extends Vue {
   async refetchAmalgamatingBusinessesInfo (): Promise<void> {
     const fetchTingInfo = async (item: any): Promise<AmalgamatingBusinessIF> => {
       const tingBusiness = await this.fetchAmalgamatingBusinessInfo(item)
-      // *** TODO: need to improve this
-      // (no auth info means not affiliated, which may or may not be foreign)
-      if (!tingBusiness.authInfo) {
+      // no auth info and business info means foreign, otherwise LEAR (affiliated or non-affiliated)
+      if (!tingBusiness.authInfo && !tingBusiness.businessInfo) {
```

Comment:
So I've tested this extensively.

We have two cases now. If the tingBusiness does not have both authInfo and businessInfo, then it's a Foreign. If it does, then it's LEAR. However, please see next comment.